### PR TITLE
Add Vimeo and Twitter support

### LINKED ISSUE #22: IfConfirm doesnt differentiate between Youtube and Vimeo Videos
Resulting in Vimeo Videos using the same Cookie with Youtube Videos.

Will provide a PR where i've fixed this tomorrow

### PR DESCRIPTION
**Pull request by Molochem**
Fixes #22 
While working on the new Kitconcept Website, I noticed that there is no Differentiation between Vimeo and Youtube Videos cookie-wise.

I've added said Differentiation in the Video-View, and also added Twitter Conformity for Volto Social Block Support

### CODE REVIEW

**Comment by Molochem** [comment]
As mentioned in #22 , here is the PR

**Comment by davisagli** [comment]
@Molochem Please add "Fixes #22" to the description of this PR. Then when we merge the PR, the related issue will automatically close.

File: src/customizations/components/manage/Blocks/Video/View.jsx [hunk 1]
```diff
@@ -9,66 +9,117 @@ import { Embed } from 'semantic-ui-react';
 import cx from 'classnames';
 import { isInternalURL, flattenToAppURL } from '@plone/volto/helpers';
 
-import { IfConfirm } from '../../../../../components';
+import { default as IfConfirm } from '@kitconcept/volto-dsgvo-banner/components/IfConfirm/IfConfirm';
```
Comment (davisagli) [inline src/customizations/components/manage/Blocks/Video/View.jsx:12]:
```suggestion
import IfConfirm from '@kitconcept/volto-dsgvo-banner/components/IfConfirm/IfConfirm';
```

**Review by davisagli — APPROVED** [review]
@Molochem Thanks for this fix, it looks good to me.

We should probably add a full list of the supported `modules` in README.md

**Comment by davisagli** [comment]
Closing because it was combined with #24

**Pull request closed without merging**